### PR TITLE
deleting old spec file, starting transition to new testing format, an…

### PR DESCRIPTION
**Pull request by McKnight-42**
Adding the basic structure to use the new converted spec tests in pytest format

### CODE REVIEW

**Comment by McKnight-42** [comment]
@gshank @ChenyuLInx does this seem like all we would need to be able to begin to use the new tests? do we only want to use the initial converted spec tests or are there going to be other required tests from the new [basic](https://github.com/dbt-labs/dbt-core/tree/main/tests/adapter/dbt/tests/adapter/basic) file?

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[pytest]
+filterwarnings =
+    ignore:.*'soft_unicode' has been renamed to 'soft_str'*:DeprecationWarning
+    ignore:unclosed file .*:ResourceWarning
+env_files =
+    test.env
+testpaths =
+    tests
```
Comment (ChenyuLInx) [inline pytest.ini:8]: Do we need this file?

File: tests/template_test.py [hunk 2]
```diff
@@ -24,33 +24,33 @@ def test_bake_project_default_adapter_name(cookies):
...
         assert result.exit_code == 0
         assert result.exception is None
-        assert result.context['adapter_src'] =='sql'
-        assert result.context['adapter_src'] != 'base'
-        assert result.context['adapter_src'] != '1'
-        assert result.context['adapter_src'] != '2'
+        assert result.context['is_sql_adapter'] =='true'
+        assert result.context['is_sql_adapter'] != 'false'
+        assert result.context['is_sql_adapter'] != '1'
+        assert result.context['is_sql_adapter'] != '2'
```
Comment (ChenyuLInx) [inline tests/template_test.py:45]: I don't think we need to check the rest, since checking the first already confirmed what `context['is_sql_adapter']` is

**Review by ChenyuLInx — APPROVED** [review]
This looks good! I think we can test this by creating a new `dbt-snowflake` adapter repo with the scaffolding repo. Copy the actual logic code over, then run the test and look whether it works

File: pytest.ini (hunk 1 above)
Comment (McKnight-42) [reply pytest.ini:8]: I believe we had pulled this over as part of the tests fixes yesterday can test to see if it can be removed.

**Pull request merged**